lib/Models/UserKeys.js

```diff
@@ -0,0 +1,38 @@
+(function() {
+  var BaseModel, UserKeys,
+    bind = function(fn, me){ return function(){ return fn.apply(me, arguments); }; },
+    extend = function(child, parent) { for (var key in parent) { if (hasProp.call(parent, key)) child[key] = parent[key]; } function ctor() { this.constructor = child; } ctor.prototype = parent.prototype; child.prototype = new ctor(); child.__super__ = parent.prototype; return child; },
```

Comment:
Line is too long.<br>Expected '{' and instead saw 'child'.<br>A constructor name should start with an uppercase letter.